lib/azurex/authorization/shared_key.ex

```diff
@@ -76,13 +74,9 @@ defmodule Azurex.Authorization.SharedKey do
     struct(request, headers: headers)
   end
 
-  def formatted(date_time) do
+  def formatted(%DateTime{zone_abbr: "UTC"} = date_time) do
     date_time
-    |> Timex.format!("{WDshort}, {0D} {Mshort} {YYYY} {h24}:{m}:{s} {Zname}")
-    # x-ms-date field expects `GMT`
-    |> String.replace("Etc/GMT", "GMT")
-    # Just in case we have old
-    |> String.replace("Etc/UTC", "GMT")
+    |> Calendar.strftime("%a, %d %b %Y %H:%M:%S GMT")
```

Comment:
This is a much better solution. Nice one!

Reply:
Thank you. I feel like people add everywhere `timex` prematurely.